docs/how-tos/helm-upgrade.md

```diff
@@ -6,32 +6,35 @@ using [a Helm repository hosted on GitHub](https://azure.github.io/application-g
 Before we begin the upgrade procedure, ensure that you have added the required repository:
 
 - View your currently added Helm repositories with:
+
     ```bash
     helm repo list
     ```
 
 - Add the AGIC repo with:
+
     ```bash
     helm repo add \
         application-gateway-kubernetes-ingress \
         https://azure.github.io/application-gateway-kubernetes-ingress/helm/
     ```
 
+## Upgrade
 
-### Upgrade
-1) Refresh the AGIC Helm repository to get the latest release:
+1. Refresh the AGIC Helm repository to get the latest release:
 
     ```bash
     helm repo update
     ```
 
-2) View available versions of the `application-gateway-kubernetes-ingress` chart:
+1. View available versions of the `application-gateway-kubernetes-ingress` chart:
 
-    ```bash
+    ``` bash
```

Comment:
It seems the we accidentally added a space here.
Would this lead to the word `bash` appearing in the rendered document?

Reply:
let me check that once the docs are generated